csrc/device_lower/pass/allocation.cpp

```diff
@@ -345,6 +347,16 @@ class AllocationInserter : public kir::ExprMutator {
       auto alloc_expr = createAllocExpr(allocation, is_output);
       auto init_expr = createInitExpr(allocation, init);
 
+      // Find the largest circular buffer depth; Used for tma buffer allocation
```

Comment:
Should this be: `// Check that all circular buffer depth match`

Reply:
I added the check but didn't updated the comment.